distro/run/core/src/main/java/org/camunda/bpm/run/CamundaBpmRunProcessEngineConfiguration.java

```diff
@@ -60,4 +63,12 @@ protected void initTelemetryData() {
     Set<String> camundaIntegration = telemetryData.getProduct().getInternals().getCamundaIntegration();
     camundaIntegration.add(CamundaIntegration.CAMUNDA_BPM_RUN);
   }
+
+  protected void configureProcessEnginePlugins(List<ProcessEnginePlugin> processEnginePlugins, CamundaBpmRunProperties runProperties) {
```

Comment:
This function encapsulates config logic which is scattered in `master` in `CamundaBpmRunProcessEngineConfiguration`